pkg/cmd/server/api/v1/conversions.go

```diff
@@ -219,6 +219,15 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 				obj.ServiceAccountMethod = "prompt"
 			}
 		},
+		func(obj *AdmissionConfig) {
```

Comment:
This is defaulting, the defaulting functions just got added in the conversions file. I can move them all if we want to